### PR TITLE
use juju cards + lint cloud.css

### LINKED ISSUE #829: Use juju-cards for /cloud/juju
## Summary
There is a bundle section on the juju page which displays a list of cards previewing bundles. These should be replaced with juju-cards.

## Process
- Replace each div with the juju-card div and a single include of the script above them
- Use https://jujucharms.com/community/cards to help

### Current
- Hard coded bundle cards

### Expected
- Use the juju-card project to load the data from the API

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* have /cloud/juju using the embeddable card
* linted and cleaned up cloud.css a lot - mostly removing un-used css and in-lining media queries
* fixed some broken class names and css on the training page for the hero image
* removed some share partials that are no longer used

## QA

1. go to /cloud/juju and see that it is using the juju cards and look good
2. go to /cloud/training and see that the medals look better
3. go through the rest of the cloud section and see that it's the same as it was

## Issue / Card

Fixed #829

### CODE REVIEW

File: static/css/section/_cloud.scss [hunk 1]
```diff
@@ -191,1234 +223,479 @@
...
+    dt {
+      float: left;
+      margin-right: 6px;
 
-.nav-list-features ul {
-  margin-right: -$gutter-width;
-  position: relative;
-}
+      &::after {
+        content: ": ";
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: Single quotes please

File: static/css/section/_cloud.scss [hunk 2]
```diff
@@ -191,1234 +223,479 @@
...
+  }
 
-.box-cloud-contact .box-cloud-contact-header p {
-  color: #656565; // DDD design issue?
-  font-size: 1.3125em;
-  margin-bottom: 0;
-  line-height: 1.4;
-  margin: 0;
+  h1 {
+    background-image: url("#{$asset-server}33b96a7f-maas-logo.svg");
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: Single quotes please

File: static/css/section/_cloud.scss [hunk 3]
```diff
@@ -191,1234 +223,479 @@
...
+// no-svg and opera-mini fixes for cloud/maas
+.no-svg,
+.opera-mini {
 
-.box-try-cloud-guest p {
-  border-bottom: 1px dotted $warm-grey;
-  margin-bottom: 20px;
-  padding-bottom: 20px;
+  .cloud-maas h1 {
+    background-image: url("#{$asset-server}6574f6c8-image-picto-maas-40.png");
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: Single quotes please

**Comment by anthonydillon** [comment]
## QA
/cloud/juju
- The juju cards work well, do you think they are to boxey? Maybe nice to remove the local `.juju-bundle-card` styling.

/cloud/lxd
- The code snippet in "Getting started with LXD" section is white not dark background

### Small screen 
/cloud/public-cloud
- The "Fast service deployment with Juju" image has to much padding at the top with a border on the right
- Same with the image under the "The best management and support" section

/cloud/partners
- The cloud logo in the hero is misplaced

File: static/css/section/_cloud.scss (hunk 3 above)
Comment (pmahnke) [inline static/css/section/_cloud.scss]: done.

File: static/css/section/_cloud.scss (hunk 2 above)
Comment (pmahnke) [inline static/css/section/_cloud.scss]: done

File: static/css/section/_cloud.scss (hunk 1 above)
Comment (pmahnke) [inline static/css/section/_cloud.scss]: done

**Comment by pmahnke** [comment]
/cloud/juju - fixed... it was stupidly in local.scss
/cloud/lxd - don't see it
/cloud/partners - fixed... sort of... main cloud logo is ok, bg clouds are still there

**Comment by anthonydillon** [comment]
The code snippet is still white for me. On live its coming from _desktop.scss so may have been clean up in the desktop branch. Can you pull master tip and rebase please.

**Comment by anthonydillon** [comment]
Still see this in the cloud/public-cloud
public cloud cloud ubuntu

**Comment by anthonydillon** [comment]
That's all, two points remain.

**Comment by pmahnke** [comment]
/cloud/lxd - found the <pre> stuff.. it is in the old desktop.scss file... direct element stuff... I moved it into the .download {} section and it's better.
/cloud/public-cloud added a media query for the div right border and padding issue

**Pull request merged**